README.md

```diff
@@ -140,27 +154,32 @@ The code for the plugins are under `plugins`.
 
 ## Roadmap
 
+![image](https://github.com/user-attachments/assets/f64b51df-03be-44d1-ae35-82e6dc960d06)
+
+See how we're tracking towards major [milestones]([url](https://github.com/onlook-dev/onlook/milestones)), and read the [wiki](https://github.com/onlook-dev/onlook/wiki/Roadmap) for details on each version of Onlook. Here's a rough overview of some of the major features we're looking at:
+
 * [X] Browser
 * [X] Editor
 * [X] Write-to-code
 * [ ] Components
 * [ ] Variables
-* [ ] Code Editor
-
-See the [roadmap](https://github.com/onlook-dev/onlook/wiki/Roadmap) for a detailed list of project goals and improvements.
 
-See the [open issues](https://github.com/onlook-dev/onlook/issues) for a full list of proposed features (and known issues).
+Also check the [open issues](https://github.com/onlook-dev/onlook/issues) for a full list of proposed features (and known issues).
 
 
 ## Contributing
 
+![image](https://github.com/user-attachments/assets/ecc94303-df23-46ae-87dc-66b040396e0b)
+
 If you have a suggestion that would make this better, please fork the repo and create a pull request. You can also [open issues](https://github.com/onlook-dev/onlook/issues).
 
 
 See the [CONTRIBUTING.md](CONTRIBUTING.md) for instructions and code of conduct.
 
 ## Contact
 
+![image](https://github.com/user-attachments/assets/60684b68-1925-4550-8efd-51a1509fc953)
+
```

Comment:
Is there a way to get contributor's avatars on here? I feel like there should be
https://github.com/calcom/cal.com?tab=readme-ov-file#contributors